.github/workflows/testBuild.yml

```
@@ -17,38 +17,33 @@ jobs:
   validateActor:
     runs-on: ubuntu-latest
     outputs:
-      READY_TO_BUILD: ${{steps.readyToBuild.outputs.READY_TO_BUILD}}
+      READY_TO_BUILD: ${{ fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && fromJSON(steps.hasReadyToBuildLabel.outputs.HAS_READY_TO_BUILD_LABEL) }}
     steps:
       - id: isUserTeamMember
         uses: tspascoal/get-user-teams-membership@baf2e6adf4c3b897bd65a7e3184305c165aec872
         with:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           username: ${{ github.actor }}
           team: expensify-expensify
-      - name: Remove label if it was added by an unauthorized user
-        if: ${{ !fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && github.event.label.name == 'Ready To Build' }}
-        uses: actions-ecosystem/action-remove-labels@v1
-        with:
-          labels: 'Ready To Build'
-      - name: Throw exception if label was added by an unauthorized user
-        if: ${{ !fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && github.event.label.name == 'Ready To Build' }}
-        run: |
-          echo "The 'Ready To Build' label was added by an unauthorized user"
-          exit 1
-      - id: readyToBuild
-        name: Set READY_TO_BUILD flag
-        run: echo "READY_TO_BUILD=${{ fromJSON(steps.isUserTeamMember.outputs.isTeamMember) || contains(github.event.pull_request.labels.*.name, 'Ready To Build') }}" >> "$GITHUB_OUTPUT"
+
+      - id: hasReadyToBuildLabel
+        name: Set HAS_READY_TO_BUILD_LABEL flag
+        run: echo "HAS_READY_TO_BUILD_LABEL=$(gh pr view "$PULL_REQUEST_NUMBER" --json labels --jq '.labels[].name' | grep -q 'Ready To Build' && echo 'true')" >> "$GITHUB_OUTPUT"
```

Maybe missed this suggestion:

```suggestion
        run: echo "HAS_READY_TO_BUILD_LABEL=$(gh pr view "${{ env.PULL_REQUEST_NUMBER }}" --json labels --jq '.labels[].name' | grep -q 'Ready To Build' && echo 'true')" >> "$GITHUB_OUTPUT"
```

Reason being – I'm not sure if putting `PULL_REQUEST_NUMBER` in the environment actually makes it available as a regular bash variable